pkgs/by-name/cu/cups-kyocera-3500-4500/package.nix

```diff
@@ -0,0 +1,86 @@
+{ lib
+, stdenv
+, fetchzip
+, cups
+, autoPatchelfHook
+, python3Packages
+
+# Sets the default paper format: use "EU" for A4, or "Global" for Letter
+, region ? "EU"
+}:
+
+assert region == "Global" || region == "EU";
+
+let
+  kyodialog_version_short = "9";
```

Comment:
Looking at the other Kyocera drivers, it seems the precedent is for them to use `9` here when the full version is `9.0`, and to use `9.1` when the full version is `9.1`. Given that, I'm inclined to leave this as is, and treat this as effectively an arbitrary string that'll need updating to a different arbitrary string for future releases, rather than implying it can be predictably computed.

That said, I don't have very strong feelings about this; I'm happy to make the change if that'd still be preferable.